grakn-graql/src/test/java/ai/grakn/graql/internal/query/InsertQueryTest.java

```diff
@@ -224,30 +223,30 @@ public void testInsertOntology() {
                 var().rel("evolves-from", "y").rel("evolves-to", "z").isa("evolution")
         ).execute();
 
-        assertTrue(qb.match(label("pokemon").sub(Schema.MetaSchema.ENTITY.getLabel().getValue())).ask().execute());
-        assertTrue(qb.match(label("evolution").sub(Schema.MetaSchema.RELATIONSHIP.getLabel().getValue())).ask().execute());
-        assertTrue(qb.match(label("evolves-from").sub(Schema.MetaSchema.ROLE.getLabel().getValue())).ask().execute());
-        assertTrue(qb.match(label("evolves-to").sub(Schema.MetaSchema.ROLE.getLabel().getValue())).ask().execute());
-        assertTrue(qb.match(label("evolution").relates("evolves-from").relates("evolves-to")).ask().execute());
-        assertTrue(qb.match(label("pokemon").plays("evolves-from").plays("evolves-to")).ask().execute());
+        assertTrue(qb.match(label("pokemon").sub(Schema.MetaSchema.ENTITY.getLabel().getValue())).iterator().hasNext());
+        assertTrue(qb.match(label("evolution").sub(Schema.MetaSchema.RELATIONSHIP.getLabel().getValue())).iterator().hasNext());
+        assertTrue(qb.match(label("evolves-from").sub(Schema.MetaSchema.ROLE.getLabel().getValue())).iterator().hasNext());
+        assertTrue(qb.match(label("evolves-to").sub(Schema.MetaSchema.ROLE.getLabel().getValue())).iterator().hasNext());
+        assertTrue(qb.match(label("evolution").relates("evolves-from").relates("evolves-to")).iterator().hasNext());
+        assertTrue(qb.match(label("pokemon").plays("evolves-from").plays("evolves-to")).iterator().hasNext());
 
         assertTrue(qb.match(
```

Comment:
Possible Nitpick: Same for these guys, maybe it would be better to use Iterables.getOnlyElement(...)